PizzaBuilder/PizzaBuilder/Program.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PizzaBuilder
+{
+    public interface IPizzaBuilder
+    {
+        void Reset();
+        void buildDough();
+        void grabPan();
+        void buildSauce();
+        void buildToppings();
+
+    }
+    class DeepdishPizzaBuilder : IPizzaBuilder
+    {
+        private DeepdishPizza pizza = new DeepdishPizza();
+
+        public DeepdishPizzaBuilder()
+        {
+            this.Reset();
+        }
+        public void Reset()
```

Comment:
The `reset` method is empty, I think you forgot to implement it..
it should initialize new `DeepdishPizza` object.